extensions/jsonld-serdes/src/main/java/org/eclipse/dataspaceconnector/serializer/JsonLdService.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2022 Fraunhofer Institute for Software and Systems Engineering
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Fraunhofer Institute for Software and Systems Engineering - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.serializer;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.eclipse.dataspaceconnector.serializer.types.calendar.XmlGregorianCalendarModule;
+
+import java.text.SimpleDateFormat;
+
+/**
+ * Provides a customized {@link ObjectMapper}.
+ */
+public final class JsonLdService {
```

Comment:
I would call it only `JsonLd`, as this is not a service (as intended in the EDC language)